components/edit-collective/EditVirtualCardModal.js

```diff
@@ -0,0 +1,261 @@
+            <StyledInputField
+              gridColumn="1/3"
+              labelFontSize="13px"
+              label={<FormattedMessage defaultMessage="Which user will be responsible for this card?" />}
+              htmlFor="assignee"
+              error={formik.touched.assignee && formik.errors.assignee}
+            >
+              {inputProps => (
+                <CollectivePicker
+                  {...inputProps}
+                  name="assignee"
+                  id="assignee"
+                  groupByType={false}
+                  collectives={collectiveUsers}
+                  collective={formik.values.assignee}
+                  isDisabled={isLoadingUsers || isBusy}
+                  onChange={option => formik.setFieldValue('assignee', option.value)}
+                />
+              )}
+            </StyledInputField>
+
+            <StyledInputField
+              gridColumn="1/3"
+              labelFontSize="13px"
+              label={<FormattedMessage defaultMessage="Card name" />}
+              htmlFor="cardName"
+              error={formik.touched.cardName && formik.errors.cardName}
+            >
+              {inputProps => (
+                <StyledInput
+                  {...inputProps}
+                  name="cardName"
+                  id="cardName"
+                  onChange={formik.handleChange}
+                  value={formik.values.cardName}
+                  disabled={isBusy}
+                  guide={false}
+                />
+              )}
+            </StyledInputField>
+
+            {canEditMonthlyLimit && (
+              <StyledInputField
+                gridColumn="1/3"
+                labelFontSize="13px"
+                label={<FormattedMessage defaultMessage="Monthly limit" />}
+                htmlFor="monthlyLimit"
+                error={formik.touched.monthlyLimit && formik.errors.monthlyLimit}
+              >
+                {inputProps => (
+                  <StyledInputAmount
+                    {...inputProps}
+                    id="monthlyLimit"
+                    currency={virtualCard.currency}
+                    prepend={virtualCard.currency}
+                    onChange={value => formik.setFieldValue('monthlyLimit', value)}
+                    value={formik.values.monthlyLimit}
+                    disabled={isBusy}
+                  />
+                )}
+              </StyledInputField>
+            )}
+          </Grid>
+        </ModalBody>
+        <ModalFooter isFullWidth>
+          <Container display="flex" justifyContent={['center', 'flex-end']} flexWrap="Wrap">
+            <StyledButton
+              my={1}
+              minWidth={140}
+              buttonStyle="primary"
+              data-cy="confirmation-modal-continue"
+              loading={isBusy}
+              type="submit"
+              textTransform="capitalize"
+            >
+              <FormattedMessage defaultMessage="Edit virtual card" />
```

Comment:
Small suggestion. Maybe use something like `Confirm` or `Confirm Changes` I think? 🤔 Because the modal is already to "edit virtual card" so when pressing this button what we are doing is confirming, not editing it. 

```suggestion
              <FormattedMessage defaultMessage="Confirm" />
```

Reply:
"Update" as button perhaps.